badges/badges.md

```diff
@@ -1,26 +1,25 @@
-# Badges!
+# Badges
```

Comment:
The exclamation mark stays unless you have a very good reason to drop it

Reply:
OK, got it

Reply:
fixed